packages/mobile/src/components/withdraw-usdc-drawer/WithdrawUSDCDrawer.tsx

```diff
@@ -114,22 +127,38 @@ const WithdrawUSDCForm = ({ onClose }: { onClose: () => void }) => {
       formPage = <TransferSuccessful onDone={onClose} />
       break
     default:
-      formPage = <EnterTransferDetails />
+      formPage = (
+        <EnterTransferDetails
+          scrollViewRef={scrollViewRef}
+          balanceNumberCents={balanceNumberCents}
+        />
+      )
       break
   }
 
   return (
-    <Flex gap='xl' pb='xl' ph='l'>
-      <Flex gap='l'>
-        <Flex row justifyContent='center' pt='xl'>
-          <Text variant='label' strength='strong' size='xl' color='default'>
-            {walletMessages.withdrawCash}
-          </Text>
+    <BottomSheetScrollView
```

Comment:
Oh interesting so bottom sheet is the move instead of drawer? You think we should consider reimplementing drawer with bottom sheet?

Reply:
yeah potentially!